### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by clairedelaleau**
Links and icons added

### CODE REVIEW

**Review by BBriset — DISMISSED** [review]
ok according to me

File: src/components/footer/githubMobile.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  {
+    link: 'https://github.com/WildCodeSchool/p2-g2-sept21-lille-react-Videotheque',
+    memberName: 'KeepN Team',
+  },
+];
+
+const GithubMobile = () => {
+  return (
+    <div className="teamProfile">
+      {team.map((e) => (
```
Comment (Hacene010) [inline src/components/footer/githubMobile.jsx:13]: Why did you use a map of just one team ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]
One thing to think about. Plz see the comment.

**Review by BBriset — DISMISSED** [review]
ok to me

File: src/components/footer/Credits.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { SiThemoviedatabase } from 'react-icons/si';
+
+const Credits = () => {
+  return (
+    <div className="wrapper">
+      <div className="icon credits">
```
Comment (Hacene010) [inline src/components/footer/Credits.jsx:6]: Are these two classNames or just one ?

File: src/components/footer/footer.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  box-shadow: 0 10px 10px rgba(0, 0, 0, 0.1);
+  cursor: pointer;
+  transition: all 0.2s cubic-bezier(0.68, -0.55, 0.265, 1.55);
+}
+
+.wrapper .toolTip {
+  display: visible;
```
Comment (Hacene010) [inline src/components/footer/footer.css]: Does visible really exist ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Seen one evil.

File: src/components/footer/footer.css (hunk 3 above)
Comment (clairedelaleau) [reply src/components/footer/footer.css]: Thanks for this comment. It's a mistake. I fix it.

**Review by BBriset — DISMISSED** [review]

File: src/components/footer/githubMobile.jsx (hunk 1 above)
Comment (LoicBrassart) [inline src/components/footer/githubMobile.jsx:13]: This conversation is not close, is it ? You don't `.map()` a one-element-array, it makes no sense ;-)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Still an Evil to fix

File: src/components/footer/githubMobile.jsx (hunk 1 above)
Comment (clairedelaleau) [reply src/components/footer/githubMobile.jsx:13]: This conversation is closed. Yes, it makes no sense :) This component is deleted.

Comment (clairedelaleau) [reply src/components/footer/githubMobile.jsx:13]: Yes, this conversation is closed. It makes no sense, off course :) This component is deleted.

**Comment by LoicBrassart** [comment]
No SM validation

**Review by BBriset — APPROVED** [review]

File: src/components/FicheFilm/FicheFilm.css [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+.posterTrailer {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-top: 41px;
+  margin-bottom: 40px;
+}
+
+.backgroundPoster {
+  z-index: -1;
```
Comment (LoicBrassart) [inline src/components/FicheFilm/FicheFilm.css]: ;_;

File: src/components/FicheFilm/FicheFilm.css [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+
+    .posterTrailer > img {
+      margin-bottom: 25px;
+      width: 59%;
+      height: 59%;
+      box-shadow: 20px 10px 23px -10px rgba(0, 0, 0, 0.384);
+    }
+
+    .backgroundPoster {
+      z-index: -1;
```
Comment (LoicBrassart) [inline src/components/FicheFilm/FicheFilm.css]: Re-;_;

File: src/components/FicheFilm/FicheFilm.jsx [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  let UsersScorePictures;
+  if (voteAverages === 10) {
+    UsersScorePictures = star5;
+  } else if (voteAverages < 10 && voteAverages >= 8) {
+    UsersScorePictures = star4;
+  } else if (voteAverages < 8 && voteAverages >= 6) {
+    UsersScorePictures = star3;
+  } else if (voteAverages < 6 && voteAverages >= 4) {
+    UsersScorePictures = star2;
+  } else if (voteAverages < 4) {
```
Comment (LoicBrassart) [inline src/components/FicheFilm/FicheFilm.jsx]: Simplify your conditions
I already checked this code: you probalby created branches rom something other than `dev`, giving me more work to do for nothing -_-

File: src/components/FicheFilm/FicheFilm.jsx [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    UsersScorePictures = star1;
+  }
+
+  const hour = (runtimes - (runtimes % 60)) / 60;
+  const min = (runtimes / 60 - hour) * 60;
```
Comment (LoicBrassart) [inline src/components/FicheFilm/FicheFilm.jsx]: Use `%`

File: src/components/carousel/Carousel.jsx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+  useEffect(() => {
+    setProperties({
+      ...properties,
+      slidesToShow: window.matchMedia('(min-width: 780px)').matches ? 5 : 1,
+    });
+    axios
+      .get(
+        'https://api.themoviedb.org/3/movie/popular?api_key=599ded6f0fc3bcaee1882e83ae0d438a&language=en-US&page=1'
+      )
+
```
Comment (LoicBrassart) [inline src/components/carousel/Carousel.jsx]: Useless empty line

File: src/components/carousel/Carousel.jsx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+        setMoviePics(data.results);
+      });
+  }, [window.innerWidth]);
+
+  useEffect(() => {
+    axios
+      .get(
+        'https://api.themoviedb.org/3/movie/popular?api_key=599ded6f0fc3bcaee1882e83ae0d438a&language=en-US&page=1'
+      )
+
```
Comment (LoicBrassart) [inline src/components/carousel/Carousel.jsx]: Useless empty line

File: src/components/carousel/carousel.css [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+    display: flex;
+    justify-content: center;
+    margin-top: 10vh;
+  }
+}
+
+/* .imgContain:hover {
+  -ms-transform: scale(1.5);
+  transform: scale(1.5);
+} */
```
Comment (LoicBrassart) [inline src/components/carousel/carousel.css]: Dead code

File: src/components/eye/Eye.jsx [hunk 11]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './eye.css';
+
+const Eye = () => {
+  return (
+    <>
```
Comment (LoicBrassart) [inline src/components/eye/Eye.jsx]: Useless Fragment

File: src/components/footer/Footer.jsx [hunk 12]
```diff
@@ -1,11 +1,14 @@
...
+
 import Credits from './Credits';
 import Team from './Team';
 import SocialMedia from './SocialMedia';
+
 import './footer.css';
 
 const Footer = () => {
   return (
+
```
Comment (LoicBrassart) [inline src/components/footer/Footer.jsx:11]: Useless empty line

File: src/components/footer/Footer.jsx [hunk 13]
```diff
@@ -43,6 +46,8 @@ const Footer = () => {
         </p>
       </div>
     </section>
+
+
```
Comment (LoicBrassart) [inline src/components/footer/Footer.jsx:50]: Useless empty lines + I'm pretty sure Prettier didn't validate this, what happened ?

File: src/components/footer/footer.css [hunk 14]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (LoicBrassart) [inline src/components/footer/footer.css:1]: Useless empty line

File: src/components/header/Header.css [hunk 15]
```diff
@@ -0,0 +1,164 @@
...
+      #f1bc28
+    ),
+    linear-gradient(
+      60deg,
+      #f7d067 40%,
+      transparent 25.5%,
+      transparent 75%,
+      #f5a81b 75%,
+      #f1bc28
+    );
```
Comment (LoicBrassart) [inline src/components/header/Header.css]: Are you applying all of them on the same element ? Is every one ofthem bypassing the previous one ?

File: src/components/header/Header.css [hunk 16]
```diff
@@ -0,0 +1,164 @@
...
+  .navBar {
+    height: 10vh;
+    width: 100%;
+    background: rgba(196, 196, 196, 0.2);
+    display: flex;
+    align-items: center;
+    flex-flow: row, nowrap;
+    justify-content: space-between;
+
+    /* box-shadow: 0 3px 10px 1px rgba(0, 0, 0, 0.08); */
```
Comment (LoicBrassart) [inline src/components/header/Header.css]: Dead code

File: src/components/searchBar/SearchBar.jsx [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+const SearchBar = () => {
+  const [query, setQuery] = useState('');
+  const [results, setResults] = useState([]);
+
+  const onChange = (e) => {
+    setQuery(e.target.value);
+    Axios.get(
+      `https://api.themoviedb.org/3/search/movie?api_key=599ded6f0fc3bcaee1882e83ae0d438a&language=en-US&page=1&include_adult=false&query=${e.target.value}`
+    )
+      .then((response) => response.data)
```
Comment (LoicBrassart) [inline src/components/searchBar/SearchBar.jsx]: Nope

File: src/components/searchBar/SearchBar.jsx [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+      .then((data) => setResults(data.results))
```
Comment (LoicBrassart) [inline src/components/searchBar/SearchBar.jsx]: `.then(({data)}` to make it work out of the box

File: package.json [hunk 19]
```diff
@@ -8,9 +8,16 @@
...
+    "react-burger-menu": "^3.0.6",
     "react-dom": "^17.0.2",
+    "react-router": "^6.0.0",
+    "react-router-dom": "^6.0.0",
     "react-scripts": "4.0.3",
-    "router": "^1.3.5"
+    "react-slick": "^0.28.1",
+    "react-slideshow-image": "^3.6.0",
+    "router": "^1.3.5",
+    "slick-carousel": "^1.8.1"
```
Comment (LoicBrassart) [inline package.json]: For several of these packags, I want you to check you use them (some look like errors to me, but I may be wrong)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils to fix

**Pull request closed without merging**